app/assets/javascripts/discourse/app/lib/sidebar/common/community-section/about-section-link.js

```diff
@@ -16,10 +16,13 @@ export default class AboutSectionLink extends BaseSectionLink {
   }
 
   get text() {
-    return I18n.t("sidebar.sections.community.links.about.content");
+    return I18n.t(
+      `sidebar.sections.community.links.${this.overridenName.toLowerCase()}.content`,
+      { defaultValue: this.overridenName }
+    );
   }
 
   get prefixValue() {
-    return "info-circle";
+    return this.overridenIcon || "info-circle";
```

Comment:
I think it'll be useful to introduce a pattern in the parent class such that child links will only have to set the default text and default prefix value being used. Something along the lines of:

```
class BaseCommunitySectionLink {
  get prefixValue() {
    return this.overridenIcon || this.defaultPrefixValue()
  }
}
```

Otherwise, it makes the parent class harder to inherit from since there is "hidden" knowledge that `prefixValue` needs to return `this.overridenIcon`